### PR TITLE
en/news/blog/2022/autoscaler_tuning: add blog post about autoscaler tuning recommendations

### PR DESCRIPTION
**Pull request by ljflores**
Signed-off-by: Laura Flores <lflores@redhat.com>

### CODE REVIEW

**Comment by ljflores** [comment]
Rendered page: https://wip-autoscaler-blog-post.ceph.io/en/news/blog/2022/autoscaler_tuning/

File: src/en/news/blog/2022/autoscaler_tuning/index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  - "management"
+  - "ceph"
+  - "performance"
+---
+
+## Introduction
+
+The pg\_autoscaler module, first introduced in the Nautilus (14.2.x) release, is an excellent way to automatically manage your Ceph cluster. Based on expected pool usage, the pg\_autoscaler can make recommendations and adjust the number of placement groups in a cluster based on pool usage and tunings set by the user.
+
+The autoscaler comes with a selection of tuning options, including the `--bulk` flag, `pg_num`, `target_size_ratio`, `noautoscale`, and `bias`. In preparation for the latest Quincy release, we put these tunings through release criteria testing and derived tuning recommendations and troubleshooting tips that users can reference as they upgrade to Quincy. In this blog post, we summarize initial challenges posed by the autoscaler, outline best tuning practices that came as a result of release criteria testing, and end with some autoscaler troubleshooting tips.
```
Comment (vumrao) [inline src/en/news/blog/2022/autoscaler_tuning/index.md:21]: maybe we should also mention pg_num_min and pg_num_max.

File: src/en/news/blog/2022/autoscaler_tuning/index.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+To check the target ratio for a pool, use `osd pool autoscale-status` and look under the `TARGET RATIO` column. The `target_size_ratio` can be set on an existing pool with `ceph osd pool set <pool_name> target_size_ratio <ratio>`. To set the `target_size_ratio` during pool creation, use `ceph osd pool create --target-size-ratio <ratio>`.
+
+### Use the noautoscale flag to globally toggle the autoscaler for all pools.
+
+The autoscaler can now be turned on and off globally for all pools with the `noautoscale` flag. By default, this flag is set to false, and the default `pg_autoscale` mode, which is `on`, remains the same for each pool.
+
+If you want to turn the autoscaler on for every pool, use `ceph osd pool set noautoscale`. If you want to turn off the autoscaler for every pool, use `ceph osd pool unset noautoscale`. To get the current value of the noautoscale flag, use `ceph osd pool get noautoscale`.
```
Comment (vumrao) [inline src/en/news/blog/2022/autoscaler_tuning/index.md:53]: the statement should be opposite to what it has now as the flag name is *no*autoscale :)

**Comment by vumrao** [comment]
thank you @ljflores great work :+1: , other than some nits it looks good to me but I will have @neha-ojha and @kamoltat to have +1. Thank you again :)

File: src/en/news/blog/2022/autoscaler_tuning/index.md (hunk 1 above)
Comment (kamoltat) [reply src/en/news/blog/2022/autoscaler_tuning/index.md:21]: yep that make sense

File: src/en/news/blog/2022/autoscaler_tuning/index.md (hunk 2 above)
Comment (kamoltat) [reply src/en/news/blog/2022/autoscaler_tuning/index.md:53]: whoops yep, you're right.

File: src/en/news/blog/2022/autoscaler_tuning/index.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  - "tuning"
+  - "troubleshooting"
+  - "management"
+  - "ceph"
+  - "performance"
+---
+
+## Introduction
+
+The pg\_autoscaler module, first introduced in the Nautilus (14.2.x) release, is an excellent way to automatically manage your Ceph cluster. Based on expected pool usage, the pg\_autoscaler can make recommendations and adjust the number of placement groups in a cluster based on pool usage and tunings set by the user.
```
[the remark this answers is not in the document]
Comment (kamoltat) [reply src/en/news/blog/2022/autoscaler_tuning/index.md:19]: nit: s/manage your Ceph cluster/manage placement groups in your Ceph cluster.

**Review by kamoltat — APPROVED** [review]
Just added my comment, other than that, looks good!

**Comment by ljflores** [comment]
Latest rendered page: https://wip-autoscaler-blog-post.ceph.io/en/news/blog/2022/autoscaler_tuning/

**Comment by ljflores** [comment]
I will update the date once everything is finalized.

**Review by vumrao — APPROVED** [review]

**Review by neha-ojha — COMMENTED** [review]
Overall looks good, left some comments about the scale-down profile

**Comment by ljflores** [comment]
Rebased to remove all mention of the `scale-down` profile and emphasize that enabling logging is not a must. Also updated the date.

**Review by neha-ojha — APPROVED** [review]

**Pull request merged**